Packs/rasterize/Integrations/rasterize/rasterize_test.py

```diff
@@ -670,3 +672,44 @@ def test_delete_row_with_old_chrome_configurations_from_info_file():
     """.strip()
 
     assert expected_mock_file_content == mock_file_content_edited
+
+
+def test_get_chrome_browser_error(mocker: MockerFixture):
+    """
+    Given   A mailto URL is called.
```

Comment:
Where is the call to mailto URL? I think you should fix the docstring

Reply:
correct, will fix